regexes/device/mobiles.yml

```diff
@@ -2155,17 +2180,17 @@ Kazam:
 
 # Kiano
 Kiano:
-  regex: 'Kiano|Elegance_5_0|CORE 10.1 DUAL 3G|Intelect103G|SlimTab7[_ ]3GR[);/ ]'
+  regex: 'Kiano|Elegance_5_0|CORE 10.1 DUAL 3G|Intelect ?(7|8|10)|Slim ?Tab ?\d+(?:[_ ]3GR?)?[);/ ]'
   device: 'smartphone'
   models:
-    - regex: 'SlimTab7[_ ]3GR'
-      model: 'SlimTab7 GR'
+    - regex: 'Slim ?Tab ?(\d+)(?:[_ ]3GR?)?'
+      model: 'Slim Tab $1'
```

Comment:
Do you ignore the `3GR` on purpose? Or should we use `Slim ?Tab ?(\d+)([_ ]3GR?)?` / `Slim Tab $1 $2` so it would be added if available?